.circleci/config.yml

```diff
@@ -141,7 +141,6 @@ jobs:
               -secondary-kubeconfig="$secondary_kubeconfig" \
               -debug-directory="$TEST_RESULTS/debug" \
               -consul-k8s-image=hashicorpdev/consul-k8s:crd-controller-base-latest \
-              -consul-image=hashicorpdev/consul
```

Comment:
This was breaking our tests because it was setting the image for enterprise tests too (which need consul-ent)